utils/tests.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/env bash
 
+set -e
 # `SECRET_PROJECT_ROOT` must be set before running the script.
+[ -z "${SECRET_PROJECT_ROOT}" ] && exit 1
 
-set -e
+function cleanup {
+    rm -rf "${TEST_RUN_DIR}" "${TEST_TMP_DIR}"
+    exit 0
+}
+
+trap cleanup EXIT
 
 # Running all the bats-tests in a dir with spaces:
```

Comment:
We're introducing and using a bunch of env vars here. 
Can we explain them for the reader/developer?